prototype_source/max_autotune_CPU_with_gemm_template_tutorial.rst

```diff
@@ -0,0 +1,198 @@
+Max-autotune Support on CPU with GEMM Template Tutorial
```

Comment:
GEMM template is an implementation detail. Suggest to name the title from users perspective: "Use max-autotune compilation on CPU to gain further performance boost". Then in the content of the tutorial, we can talk about perf boost in GEMMs with the template implementation etc.

Reply:
Renamed to "Use max-autotune compilation on CPU to gain further performance boost"